src/AWS.Deploy.Common/ProjectDefinitionParser.cs

```diff
@@ -37,6 +37,7 @@ public async Task<ProjectDefinition> Parse(string projectPath)
         {
             if (_directoryManager.Exists(projectPath))
             {
+                projectPath = _directoryManager.GetDirectoryInfo(projectPath).FullName;
```

Comment:
- What testing you have performed?
- How can we ensure it will not break in future?
- Can this be covered via unit tests?

Reply:
Added unit tests for this. The tests will fail if the change is removed